_layouts/default.html

```diff
@@ -1,75 +1,86 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <title>{{ page.title }}</title>
 
-    <link rel="stylesheet" href="{{ site.baseurl }}/stylesheets/styles.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}/stylesheets/highlighter.css">
-    <link rel="icon" type="image/png" href="{{ site.baseurl }}/images/cpr-favicon.png">
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-    <!--[if lt IE 9]>
+<head>
```

Comment:
Did you reformat this file?

Reply:
Ah, I open the default HTML formatter setting in the VSCode. 
I will fix it.